File: specification/confluent/resource-manager/Microsoft.Confluent/stable/2024-02-13/examples-java/Organization_GetEnvironmentById.java

/**
 * Samples for Organization GetEnvironmentById.
 */
public final class Main {
    /*
     * x-ms-original-file: specification/confluent/resource-manager/Microsoft.Confluent/stable/2024-02-13/examples/
     * Organization_GetEnvironmentById.json
     */
    /**
     * Sample code: Organization_GetEnvironmentById.
     * 
     * @param manager Entry point to ConfluentManager.
     */
    public static void organizationGetEnvironmentById(com.azure.resourcemanager.confluent.ConfluentManager manager) {
        manager.organizations().getEnvironmentByIdWithResponse("myResourceGroup", "myOrganization", "dlz-f3a90de",
            com.azure.core.util.Context.NONE);
    }
}

File: specification/confluent/resource-manager/Microsoft.Confluent/stable/2024-02-13/examples-js/Organization_GetEnvironmentById.js
const { ConfluentManagementClient } = require("@azure/arm-confluent");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Get Environment details by environment Id
 *
 * @summary Get Environment details by environment Id
 * x-ms-original-file: specification/confluent/resource-manager/Microsoft.Confluent/stable/2024-02-13/examples/Organization_GetEnvironmentById.json
 */
async function organizationGetEnvironmentById() {
  const subscriptionId =
    process.env["CONFLUENT_SUBSCRIPTION_ID"] || "00000000-0000-0000-0000-000000000000";
  const resourceGroupName = process.env["CONFLUENT_RESOURCE_GROUP"] || "myResourceGroup";
  const organizationName = "myOrganization";
  const environmentId = "dlz-f3a90de";
  const credential = new DefaultAzureCredential();
  const client = new ConfluentManagementClient(credential, subscriptionId);
  const result = await client.organization.getEnvironmentById(
    resourceGroupName,
    organizationName,
    environmentId,
  );
  console.log(result);
}
